centreon/www/front_src/src/federatedModules/Load/index.tsx

```diff
@@ -44,10 +46,10 @@ export const Remote = ({
               module: component,
               remoteEntryFileName: remoteEntry,
               scope: moduleFederationName,
-              url: `./${prefix}/${moduleName}/static`
+              url: remoteUrl ?? `./${prefix}/${moduleName}/static`
```

Comment:
why?
It gonna break Widgets

Reply:
oh I see sorry

Reply:
This will enable remote federated modules with the url type "https:://centreon.analytics" or "http://localhost", the remoteUrl key is not required but it will be used as a priority if it exists, otherwise not.